solutions/dedicated/README.md

```diff
@@ -0,0 +1,60 @@
+# Deployable Architecture (DA) for the dedicated IBM Cloudant instance
+
+Deployable Architecture for the dedicated IBM Cloudant instance which deploys the following resources:
+
+- A resource group, if one is not passed in.
+- A dedicated cloudant instance
+- A basic database in the Cloudant instance
```

Comment:
By default it doesn't create any database. It supports optionally creating one, so suggest saying:

`- A Cloudant database`